src/app/planner/planner/planner.component.html

```diff
@@ -1,14 +1,13 @@
 <div class="planner-container">
-	<div class="card" *ngFor="let item of meals; let myIndex=index">
-		<div class="card-info">
-		<img class="meal-img" src='{{item.url}}' alt="meal">
-		<h2>{{item.name}}</h2>
-		</div>
-	  <div class="card-portion">
-		<h3>{{item.portion}}</h3>
-			<button id="remove-button" (click)="deleteMeals(item.id)"><h3>X</h3></button>
-		</div>
-	</div>
+	<app-card 
```

Comment:
There is a lot of styling in your planner.component.scss that is not used here, remove unused code